docs/Makefile

```diff
@@ -29,16 +29,28 @@ run_animate_demos:
 	cd . \
 
 run_goalie_demos:
+	@SKIP_DEMOS="time_partition.py gray_scott.py gray_scott_split.py"; \
 	for FILE in ${GOALIE}/demos/*.py; do \
-		cd  ${GOALIE}/demos/ && \
-		${PYTHON} $$FILE ; \
+		BASENAME=$$(basename $$FILE); \
+		if ! echo "$$SKIP_DEMOS" | grep -wq $$BASENAME; then \
+			cd ${GOALIE}/demos/ && \
+			${PYTHON} $$FILE; \
+		else \
+			echo "Skipping Goalie demo $$BASENAME"; \
+		fi; \
 	done; \
```

Comment:
Not specific to your change, but how about we simplify things slightly by only changing directory once?
```sh
cd ${GOALIE}/demos; \
for FILE in *.py; do \
  ... \
done
```
and similarly for the Animate and Movement demos.

Reply:
Nice! Done that now in the last 2 commits :)